### PR TITLE
Hatches

### PR DESCRIPTION
**Pull request by 1Sparky1**
Changes all the airlocks to be called hatches, on the player-facing end at least. Code side they're still called airlocks for the sake of not breaking everything and for ease of porting stuff. Suggested [here](https://forums.aurorastation.org/topic/16892-rename-airlocks-to-hatches).

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Comment by 1Sparky1** [comment]
!review

**Comment by alsoandanswer** [comment]
Don't think this is a good idea. We're going against the momentum of several years of them being called airlocks now, and all this does is just make things confusing.

**Review by listerla — APPROVED** [review]

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -104,8 +104,8 @@
 	update_icon()
 
 /obj/machinery/door/proc/open_hatch(var/atom/mover = null)
-	if (!hatchstate)
-		hatchstate = 1
+	if (!hatchestate)
+		hatchestate = 1
```
Comment (NonQueueingMatt) [inline code/game/machinery/doors/door.dm:108]: Hatche?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/machinery/doors/door.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/game/machinery/doors/door.dm:108]: This error is present in a few more lines too.

**Comment by 1Sparky1** [comment]
Having actually paid attention during the game, there are very few places where the name airlock is actually shown to the player. Closing.

**Pull request closed without merging**